### ISSUE TITLE
Qt display errors with no desktop environment (awesomewm + debian jessie)

### ISSUE THREAD

**Issue by d0pephish** [Vector35/binaryninja-api#504]
==Background==
My setup is Debian with Awesome Window Manger and no additional desktop environment. Typically this works great for most normal GTK apps. Never had a need to use Qt until playing with binaryninja. 

==Symptoms==
Whenever I launch binaryninja, I got a blank window with no visible buttons. 
When I switched to kde or gnome for my desktop environment, I have no issues. 

==Solution==
As it turns out, Qt5 tries to auto-detect the style to use for the windows. Because I did not have a desktop environment running, it defaults to the default xserver style, which I believe is just an empty shell. I tried to override this by using QT_STYLE_OVERRIDE env; this would change the appearance but not give full functionality. 

My solution was to add a runonce in my rc.lua to kick of gnome-settings-daemon which sets up the windowing environment for gnome. That way my env is set properly when Qt5 opens an application. 

Bottom Line: if you are having trouble launching binaryninja or other Qt5 apps in less-than-normal situations, make sure your env is set up properly.
